projects.yml

```diff
@@ -4351,12 +4369,42 @@ spec_coerce:
   categories: [Spec, Data Coercion]
   platforms: [clj, cljs]
 
+coax:
+  name: coax
+  url: https://github.com/exoscale/coax
+  categories: [Spec, Data Coercion]
+  platforms: [clj, cljs]
+
 clojure_future_spec:
   name: clojure-future-spec
   url: https://github.com/tonsky/clojure-future-spec
   categories: [Spec]
   platforms: [clj]
 
+malli:
+  name: malli
+  url: https://github.com/metosin/malli
+  categories: [Schema, Dynamic Typing, Validation, Random Data Generation]
+  platforms: [clj, cljs]
+
+minimallist:
+  name: minimallist
+  url: https://github.com/green-coder/minimallist
+  categories: [Schema, Dynamic Typing, Validation, Random Data Generation]
+  platforms: [clj, cljs]
```

Comment:
I am unsure of the `Dynamic Typing` category. I copied it from the existing entry for `spec.alpha`. What do we think?

Reply:
Dynamic typing is anything that improves runtime typing. So Spec has it because it can instrument functions and raise more sophisticated errors when the functions receive data that fails the spec.